recipe/meta.yaml

```
@@ -58,34 +58,30 @@ requirements:
     - pkg-config        # [unix]
     - pycparser
   host:
-    - openssl
-    - libffi
-    - sqlite
-    - tk
-    - zlib
     - bzip2
     - expat
-    - ncurses  # [not win]
     - gdbm     # [not win]
-    - xz
-    - tk
-    - bzip2
-    - xorg-libx11  # [linux]
     - jom ==1.1.2  # [win]
-  run:
-    - openssl
     - libffi
+    - ncurses  # [not win]
+    - openssl
     - sqlite
     - tk
+    - xorg-libx11  # [linux]
+    - xz
     - zlib
+  run:
     - bzip2
-    - ncurses  # [not win]
+    - expat
     - gdbm     # [not win]
+    - libffi
+    - ncurses  # [not win]
+    - pypy3.5 ==99999999999
```

```suggestion
    - openssl
```

It looks like the openssl run-dep got lost in the reshuffle (not sure if intentional), but the pypy3.6 pin surely should not be there...

---

Thanks, I redid the sort more carefully now.